### PR TITLE
Fix React warnings

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: styleguide.config.js [hunk 1]
```diff
@@ -34,7 +34,6 @@ module.exports = {
             'src/elements/table/body.jsx',
             'src/elements/table/row.jsx',
             'src/elements/table/cell.jsx',
-            'src/elements/table/body.jsx',
```
Comment (viktor-yakubiv) [inline styleguide.config.js:37]: Why is this changed?

Comment (Joozty) [reply styleguide.config.js:37]: it's duplicated. The body is declared three lines up. This duplicate value causes warning about the same key. (not sure why really)
